qiskit/transpiler/target.py

```diff
@@ -159,6 +159,13 @@ class Target(BaseTarget):
     angle :class:`~qiskit.circuit.library.RXGate` while ``rx`` will get the
     parameterized :class:`~qiskit.circuit.library.RXGate`.
 
+    This class can be queried via the mapping protocol, using the
+    instruction's name as a key. You can modify any property for an
+    instruction via the :meth:`.Target.update_instruction_properties` method.
```

Comment:
The `Target` here is a bit redundant because this is the class docstring here.
```suggestion
    instruction via the :meth:`.update_instruction_properties` method.
```